kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3685,6 +3695,123 @@ messages:
       return;
    }
 
+   UserGetFromContainer(items_list=$, numbers_list=$)
+   {
+      local oOldOwner, oSplit, iCan_hold, oitem, iCantGet, iNumber;
+      iCantGet = 0;
+      for oitem in items_list
+      {
+
+         iNumber = First(numbers_list);
+         oOldOwner = Send(oitem,@GetOwner);
+
+         % Make sure we're not trying to get something we've already got!
+         if oOldOwner = self
+         {
+            return;
```

Comment:
I may remove this return and the one on line 3722 and instead send a message to the user.  Right now if you trip either of these safety checks the looping over items stops and further action is aborted for safety.  Instead we could send a message like: "You reach for the %ItemName but its nowhere to be found!"  The only way I was able to force these safeties to trip is if I had two characters try to grab the same stuff simultaneously.  In that instance one character gets the stuff and the other gets a message that the item is not accessible.